app/processors/common_software_processor.py

```diff
@@ -33,12 +35,18 @@ def transform(self):
         return response.content
 
     def deliver_zip(self, zip_contents):
-        folder = get_ftp_folder(self.survey)
-        return process_zip_to_ftp(folder, zip_contents)
+        folder = self.get_ftp_folder(self.survey)
+        return self.ftp.unzip_and_deliver(folder, zip_contents)
 
     def process(self):
         zip_contents = self.transform()
         if zip_contents is None:
             return False
 
         return self.deliver_zip(zip_contents)
+
+    def get_ftp_folder(self, survey):
```

Comment:
Yeah I don't think anyone actually does was my point. Might come back in to use as part of the synthetic transaction work upcoming, it's not doing any harm there.